canvas_modules/common-canvas/src/common-properties/components/field-picker/field-picker.jsx

```diff
@@ -140,43 +140,40 @@ export default class FieldPicker extends React.Component {
 		for (let i = 0; i < fields.length; i++) {
 			const field = fields[i];
 			const columns = [];
-			let fieldContent = (
```

Comment:
This is fixing a couple issues.
1. Spacing where the icon would still take up space even when there wasn't an icon
2. Add a tooltip on all items that have an overflow
3. Only set the icon if type isn't the dmIcon value